core/src/main/scala/kafka/admin/TopicCommand.scala

```diff
@@ -191,12 +191,13 @@ object TopicCommand extends Logging {
     val topics = getTopics(zkUtils, opts)
     val reportUnderReplicatedPartitions = if (opts.options.has(opts.reportUnderReplicatedPartitionsOpt)) true else false
     val reportUnavailablePartitions = if (opts.options.has(opts.reportUnavailablePartitionsOpt)) true else false
+    val reportUnbalancedPartitions = if (opts.options.has(opts.reportUnbalancedPartitionsOpt)) true else false
     val reportOverriddenConfigs = if (opts.options.has(opts.topicsWithOverridesOpt)) true else false
```

Comment:
I think you didn't understand my comment. I am not suggesting you remove the val. I am saying that you don't need `if/else` on the right-hand side:

```scala
val reportUnderReplicatedPartitions = opts.options.has(opts.reportUnderReplicatedPartitionsOpt)
````